### DISCUSSION TITLE
How to update from 1.2.4 to 1.3.0 through rancher apps

### DISCUSSION THREAD

**Discussion by m3talstorm** [longhorn/longhorn#4127] [General]
Pretty much as the title says:

Chart Versions
100.1.2+up1.2.4 (Current)
Thu, Mar 24 2022 
100.1.1+up1.2.3
Fri, Feb 25 2022 
100.1.0+up1.2.2
Fri, Feb 25 2022 
100.0.0+up1.1.2
Fri, Feb 25 2022 

Application Version
v1.2.4

The 1.3.0 release is not listed, does a (downstream) repo need to be updated for it to appear?

**Comment by derekbit** [comment]
~~We will release 1.3.0 rancher chart recently.~~
We will release 1.3.0 rancher chart in the near future.

**Reply by m3talstorm** [reply]
Thanks for the response @derekbit , but that confused me, it has both past and future tense in the sentence :)

**Reply by c3y1huang** [reply]
You can track in https://github.com/rancher/charts/pull/1899 :slightly_smiling_face:
